src/components/NewCard/NewCard.visual.tsx

```diff
@@ -0,0 +1,54 @@
+import * as React from 'react';
+import { uniTestkitFactoryCreator } from 'wix-ui-test-utils/vanilla';
+import { NewCardWiringExample } from './docs/NewCardWiringExample';
+import { visualize, story, snap } from 'storybook-snapper';
+import { newCardDriverFactory, NewCardDriver } from './NewCard.driver';
+import { NewCardProps } from '.';
+import { onStyleProcessorDone } from '../../../test/visual/StyleProcessorUtil';
+
+const createDriver = uniTestkitFactoryCreator(newCardDriverFactory);
+const dataHook = 'storybook-e2e-NewCard';
+
+interface NewCardVisualProps {
+  newCardProps: NewCardProps;
+  onDone(TabsDriver): void;
+}
+
+class NewCardVisual extends React.Component<NewCardVisualProps> {
+  static defaultProps = {
+    newCardProps: {},
+    onDone: () => {},
+  };
+  private driver: NewCardDriver;
+
+  componentDidMount(): void {
+    this.driver = createDriver({ wrapper: document.body, dataHook });
+
+    onStyleProcessorDone()
+      .then(async () => {
+        const { onDone } = this.props as any;
+        try {
+          onDone && (await onDone(this.driver));
+        } catch (e) {}
+      })
+      .catch(() => {});
+  }
+
+  render() {
+    return (
+      <div>
+        <NewCardWiringExample />
+      </div>
+    );
+  }
+}
+
+visualize('New Card', () => {
```

Comment:
💪

Reply:
btw, since this test doesn't need to be async, we can remove the `onDone` prop from the `NewCardVisual` component.
You can see the README here: https://www.github.com/wix-incubator/storybook-snapper